### PR TITLE
[zh] Sync reference config-api file

### PR DESCRIPTION
**Pull request by mengjiao-liu**
[zh] Sync reference config-api file 
synchronize the documents of the links involved and avoid Chinese links with return code 404.

Related #27742 
- this file `content/zh/docs/reference/config-api/kube-scheduler-policy-config.v1.md`  has been synchronized in the [PR](https://github.com/kubernetes/website/pull/27742/files)

Related #27822

### CODE REVIEW

File: content/zh/docs/reference/config-api/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: Configuration APIs
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/_index.md:2]:
```suggestion
title: 配置 API
```

Comment (mengjiao-liu) [reply content/zh/docs/reference/config-api/_index.md]: Thanks for your advice. I have modified it

**Comment by tengqm** [comment]
Looks like one of the CI tasks failed to report back its status.
Let's try close and reopen.

**Comment by tengqm** [comment]
@mengjiao-liu Can you help try amend your commit to trigger a new CI check?

**Comment by mengjiao-liu** [comment]
> @mengjiao-liu Can you help try amend your commit to trigger a new CI check?

I have retriggered the CI task.

**Pull request merged**